Sources/rswift/App.swift

```diff
@@ -88,7 +88,7 @@ extension App {
             let sourceTreeURLs = SourceTreeURLs(
                 builtProductsDirURL: URL(fileURLWithPath: processInfo.environment[EnvironmentKeys.builtProductsDir] ?? EnvironmentKeys.builtProductsDir),
                 developerDirURL: URL(fileURLWithPath: processInfo.environment[EnvironmentKeys.developerDir] ?? EnvironmentKeys.developerDir),
-                sourceRootURL: URL(fileURLWithPath: processInfo.environment[EnvironmentKeys.sourceRoot] ?? EnvironmentKeys.sourceRoot),
+                sourceRootURL: URL(fileURLWithPath: processInfo.environment[EnvironmentKeys.sourceRoot] ?? ""),
```

Comment:
https://developer.apple.com/documentation/foundation/url/3126800-init

> If the path is an empty string, the system interprets it as “.”.

Reply:
Then why not write `?? "."` here? 
That way you don't need to link to the documentation to explain the code, and it does the same thing, right?

Reply:
Indeed, that might have been a better option.
Thank you for your review.